.azuredevops/modulePipelines/ms.network.frontdoorwebapplicationfirewallpolicies.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Network - FrontDoorWebApplicationFirewallPolicies'
```

Comment:
Please also add a GitHub version of the pipeline for those in the community using them :)
You can do this by copy-pasting one of the other pipelines and just update all individual values to the one of the Front Door WAF Policies module

Reply:
Added github pipeline :-)